Gemfile

```diff
@@ -16,7 +16,7 @@ gem 'kaminari',     '~> 1.1.1'
 gem 'doorkeeper',   '~> 4.2.6'
 gem 'phonelib',     '~> 0.3.6'
 gem 'twilio-ruby',  '~> 5.6.0'
-gem 'fontello_rails_converter'
```

Comment:
I think we need this for icons. Check, please

Reply:
done